boxnav/boxsim.py

```diff
@@ -1,9 +1,11 @@
 from argparse import ArgumentParser, Namespace
 from math import radians
-from pathlib import Path
+from pathlib import Path, PurePath
```

Comment:
Why `PurePath`?

Reply:
I was using it to name the automatically name datasets when using the auto_upload functionality